en/Obsidian Sync/Select files and settings to sync.md

```diff
@@ -29,34 +47,54 @@ Common examples of hidden folders that will not be synced include `.vscode`, `
+Sync settings themselves are not synced so that each each device can be separately configured according to your needs.
 
 ## Updating your synced vault settings
 
-If you have an existing multi-device sync setup and want to modify your sync settings across devices.
+If you need to modify your sync settings across multiple devices, follow these steps:
+
+> [!tip] In this context, the remote vault does not differentiate between primary and secondary devices; these terms are used for clarity.
 
 ### Primary device
 
-The primary device is your source-of-truth device. It can also mean the device you make the new change on, that you wish to sync across your other devices.
+The primary device is considered your source-of-truth. This is the device on which you make changes that you want to sync across all other devices.
 
 1. Access **Settings** → **Sync**.
 2. Activate the desired settings under **Vault configuration sync**.
-3. Restart Obsidian. On mobile or tablet, this may require a force quit.
+3. Reload or Restart Obsidian. On mobile or tablet, this may require a force quit.
 4. Allow time for the settings to synchronize with your remote vault.
 
 ### Secondary device(s)
 
-The secondary device is another device you use Obsidian with (e.g. your phone). It can also mean the devices you wish to receive an updated change on.
+The secondary device is any other device you use with Obsidian, such as your phone. It refers to devices that receive updates from the primary device.
 
 1. Navigate to **Settings** → **Sync**.
 2. Enable the desired settings under **Vault configuration sync**.
 3. Wait for the changes to be downloaded from your remote vault.
-4. Restart the application to ensure that your synchronized settings take effect. On mobile or tablet, this may require a force quit.
+4. Reload or Restart the application to ensure that your synchronized settings take effect. On mobile or tablet, this may require a force quit.
+
+## Hot reloading of settings
+
+> [!question] Wondering what Hot Reloading is? Hot reloading refreshes only the modified files without resetting the app's state, but it does not necessarily apply those changes without a reload.
+
+Obsidian has limited ability to update and apply configurations received from Sync. The following configurations will hot reload:
+
+- Most Obsidian app configurations, including hotkeys and properties.
+- Appearance configurations.
+- Configurations for already enabled Obsidian Core plugins (e.g., the location of the daily notes template folder).
+
+However, for CSS changes involving [[CSS snippets]] or [[Themes]], while the appearance configurations are received, the application must be reloaded to reflect visual changes.
+
+The following configurations will not hot reload, and will require an application reload or restart:
+
+- Graph view configurations.
+- States of Obsidian Core plugins (e.g., whether Daily Notes is enabled).
+- Community plugin configurations.
```

Comment:
I think we should remove this line. The Obsidian API provides a way for community plugins to reload when settings sync. Not all plugins are using this, but it's incorrect to say that plugin settings do not sync.

Reply:
To confirm, remove the community plugins line (92) or lines 89-92?

Reply:
"Community plugin configurations" (just line 92)